src/meshes/p4est_mesh.jl

```diff
@@ -247,15 +247,47 @@ end
                      mapping=nothing, polydeg=1, RealT=Float64,
                      initial_refinement_level=0, unsaved_changes=true)
 
-Import an uncurved, unstructured, conforming mesh from an Abaqus mesh file (`.inp`),
-map the mesh with the specified mapping, and create a `P4estMesh` from the curved mesh.
-
-Cells in the mesh file will be imported as trees in the `P4estMesh`.
-The mesh will only have one boundary `:all`, as distinguishing different boundaries
-is non-trivial.
+Main mesh constructor for the `P4estMesh` that imports an unstructured, conforming
+mesh from an Abaqus mesh file (`.inp`). Each element of the conforming mesh parsed
+from the `meshfile` is created as a [`p4est`](https://github.com/cburstedde/p4est)
+tree datatype.
+
+To create a curved unstructured mesh `P4estMesh` two strategies are available:
+
+- `p4est_mesh_from_hohqmesh_abaqus`: High-order, curved boundary information created by
+                                     [`HOHQMesh.jl`](https://github.com/trixi-framework/HOHQMesh.jl) is
+                                     available in the `meshfile`. The mesh polynomial degree `polydeg`
+                                     of the boundaries is provided from the `meshfile`. The computation of
+                                     the mapped tree coordinates is done with transfinite interpolation
+                                     with linear blending similar to [`UnstructuredMesh2D`](@ref). Boundary name
+                                     information is also parsed from the `meshfile` such that different boundary
+                                     conditions can be set at each named boundary on a given tree.
+- `p4est_mesh_from_standard_abaqus`: By default, with `mapping=nothing` and `polydeg=1`, this creates a
+                                     straight-sided from the information parsed from the `meshfile`. If a mapping
+                                     function is specified then it computes the mapped tree coordinates via polynomial
+                                     interpolants with degree `polydeg`. The mesh created by this function will only
+                                     have one boundary `:all`, as distinguishing different physical boundaries is
+                                     non-trivial.
+
+Note that the `mapping` and `polydeg` keyword arguments are only used by the `p4est_mesh_from_standard_abaqus`
+function. The `p4est_mesh_from_hohqmesh_abaqus` function obtains the mesh `polydeg` directly from the `meshfile`
+and constructs the transfinite mapping internally.
+
+The particular strategy is selected according to the header present in the `meshfile` where
+the constructor checks whether or not the `meshfile` was created with
+[`HOHQMesh.jl`](https://github.com/trixi-framework/HOHQMesh.jl).
```

Comment:
```suggestion
[HOHQMesh.jl](https://github.com/trixi-framework/HOHQMesh.jl).
```
People usually do not set Julia packages in code/monospaced font.